### PR TITLE
interop-test: disable message size check in interop-test client.

### LINKED ISSUE #3562: Many java cloud_to_cloud and cloud_to_prod interop tests failling: " No ServerStreamTracer found"
```
2017-10-09 16:53:03,682 START: cloud_to_cloud:node:csharpcoreclr_server:unimplemented_method
2017-10-09 16:53:03,686 Gradle is no longer run automatically. Make sure to run
'./gradlew installDist -PskipCodegen=true' or
'./gradlew :grpc-interop-testing:installDist -PskipCodegen=true' after any
changes. -PskipCodegen=true is optional, but requires less setup.
[jetty-alpn-agent] Using: alpn-boot-8.1.11.v20170118.jar
Running test status_code_and_message
Exception in thread "main" java.lang.AssertionError: No ServerStreamTracer found
	at io.grpc.testing.integration.AbstractInteropTest.assertServerMetrics(AbstractInteropTest.java:1828)
	at io.grpc.testing.integration.AbstractInteropTest.assertMetrics(AbstractInteropTest.java:1709)
	at io.grpc.testing.integration.AbstractInteropTest.assertMetrics(AbstractInteropTest.java:1717)
	at io.grpc.testing.integration.AbstractInteropTest.statusCodeAndMessage(AbstractInteropTest.java:1395)
	at io.grpc.testing.integration.TestServiceClient.runTest(TestServiceClient.java:282)
	at io.grpc.testing.integration.TestServiceClient.run(TestServiceClient.java:189)
	at io.grpc.testing.integration.TestServiceClient.main(TestServiceClient.java:66)

2017-10-09 16:53:03,686 TIMEOUT: cloud_to_cloud:java:java_server:status_code_and_message [pid=30665, time=180.4sec]
```

cloud_to_cloud:
https://sponge.corp.google.com/target?id=23de9377-477a-4cf9-876c-20df54eb5e83&target=github/grpc&searchFor=&show=ALL&sortBy=STATUS

cloud_to_prod:
https://sponge.corp.google.com/target?id=3fc281f5-0428-4d03-bc10-8fb9b3a7d035&target=github/grpc&searchFor=&show=ALL&sortBy=STATUS

### PR DESCRIPTION
**Pull request by zhangkun83**
The check fails in Java-Go interop test because of
https://github.com/grpc/grpc-go/issues/1572

This is also needed for the internal gRPC-GFE test.

Resolves #3562

### CODE REVIEW

**Comment by adelez** [comment]
Should this be reverted once Go tests get fixed?

File: interop-testing/src/main/java/io/grpc/testing/integration/TestServiceClient.java [hunk 1]
```diff
@@ -374,6 +374,13 @@ protected boolean serverInProcess() {
       // Server is a separate process.
       return false;
     }
+
+    @Override
+    protected boolean metricsExpected() {
+      // Exact message size doesn't match when testing with Go servers:
+      // https://github.com/grpc/grpc-go/issues/1572
```
Comment (carl-mastrangelo) [inline interop-testing/src/main/java/io/grpc/testing/integration/TestServiceClient.java]: is this a todo?

**Comment by zhangkun83** [comment]
Yes, added TODO to revert it.

**Comment by zhangkun83** [comment]
retest this please

**Pull request merged**